bin/git-authors

```diff
@@ -1,40 +1,20 @@
 #!/usr/bin/env bash
 
-LIST=false
-NO_EMAIL=false
-FILE=""
+list=0
+no_email=0
+output=""
+file=""
 
-while [[ $# -gt 0 ]]; do
-  case $1 in
-    -l|--list )
-      LIST=true
-      shift
-      ;;
-    --no-email )
-      NO_EMAIL=true
-      shift
-      ;;
-    * )
-      break
-  esac
-done
-
-if ! $LIST; then
-  FILE=$1
-  if [ -z "$FILE" ]; then
-    FILE=$(find . -mindepth 1 -maxdepth 1 \( -iname '*authors*' -o -iname '*contributors*' \) | head -n1)
-    if [ -z "$FILE" ]; then
-      FILE='AUTHORS'
-    fi
-  fi
+if [[ $# = 1 ]] && [[ "$1" = "--no-email" ]]; then
+  echo >&2 "--no-email option only can be used with --list | -l | --output"
+  exit 1
```

Comment:
Checks can be good, but I'm looking at a [previous diff](https://github.com/vanpipy/git-extras/blob/263dcb4a79fa8e21e64f9d0f6a007148a6cec9d6/bin/git-authors), and it seems like this is a breaking change? `git authors --no-email` seems to have written to file `AUTHORS`, excluding emails?

Reply:
The previous implement did not include this feature, just dump all authors has the email into `AUTHORS`. There is not breaking change to previous implement I thought now cause the behavior of `git authors AUTHORS` did not change. This condition is to forbid using `--no-email` with the default behavior when invoking `git authors` directly.

Reply:
I'm not sure I understand. On main:
```console
$ ./bin/git-authors --no-email
...
```
This branch:
```console
./bin/git-authors --no-email
--no-email option only can be used with --list | -l | --output
```
That is a breaking change.

Reply:
Also, please do not mark these conversations as resolved, the CONTRIBUTING mentions that is the responsibility of the reviewer. It makes it more difficult to track the conversation